File: lang/go/src/vac.fun/algorithms/meta/Sorting/Sorting.go
package Sorting

import (
    bubble_sort "vac.fun/algorithms/sorting/BubbleSort"
    insertion_sort "vac.fun/algorithms/sorting/InsertionSort"
    merge_sort "vac.fun/algorithms/sorting/MergeSort"
    quick_sort "vac.fun/algorithms/sorting/QuickSort"
    selection_sort "vac.fun/algorithms/sorting/SelectionSort"
)

func BubbleSort(arr []int) {
    bubble_sort.Sort(arr)
}

func InsertionSort(arr []int) {
    insertion_sort.Sort(arr)
}

func MergeSort(arr []int) {
    merge_sort.Sort(arr)
}

func QuickSort(arr []int) {
    quick_sort.Sort(arr)
}

func SelectionSort(arr []int) {
    selection_sort.Sort(arr)
}

File: lang/php/src/vac.fun/algorithms/meta/Sorting.php
<?php
    namespace vac\fun\algorithms\meta\Sorting;

    require_once "vac.fun/algorithms/sorting/BubbleSort.php";
    require_once "vac.fun/algorithms/sorting/InsertionSort.php";
    require_once "vac.fun/algorithms/sorting/MergeSort.php";
    require_once "vac.fun/algorithms/sorting/QuickSort.php";
    require_once "vac.fun/algorithms/sorting/SelectionSort.php";

    use vac\fun\algorithms\sorting\BubbleSort;
    use vac\fun\algorithms\sorting\InsertionSort;
    use vac\fun\algorithms\sorting\MergeSort;
    use vac\fun\algorithms\sorting\QuickSort;
    use vac\fun\algorithms\sorting\SelectionSort;

    function bubbleSort(array &$arr): void {
        BubbleSort\sort($arr);
    }

    function insertionSort(array &$arr): void {
        InsertionSort\sort($arr);
    }

    function mergeSort(array &$arr): void {
        MergeSort\sort($arr);
    }

    function quickSort(array &$arr): void {
        QuickSort\sort($arr);
    }

    function selectionSort(array &$arr): void {
        SelectionSort\sort($arr);
    }
?>
